scripts/get_all_extension_prs_since_last_release.sh

```diff
@@ -19,9 +19,6 @@ last_release_date=$(date -u -d"$last_release_date + 1 second" +"%Y-%m-%dT%H:%M:%
 # Initialize page number
 page=1
```

Comment:
these diffs are so we can better match the release notes patterns https://github.com/DataDog/datadog-lambda-extension/releases

Reply:
i.e. 

```
./scripts/get_all_extension_prs_since_last_release.sh
What's Changed
+ SLS-3317: Automatically instrument agent-in-container https://github.com/DataDog/datadog-agent/commit/5717a1f0c4c27a12b952c74b5b5b67890679728a
```

Reply:
Such a great idea!

Next we could even use this in `publish_prod.sh` to pre-populate the release notes. See [github docs for creating releases](https://docs.github.com/en/repositories/releasing-projects-on-github/automation-for-release-forms-with-query-parameters) and [code where we create the release](https://github.com/DataDog/datadog-lambda-extension/blob/8e3d749e32ec9cb444cf1fa68769eced6756e287/scripts/publish_prod.sh#L93-L96).